tests/integration/targets/s3_sync/tasks/main.yml

```diff
@@ -0,0 +1,116 @@
+---
+- name: set connection information for all tasks
+  set_fact:
+    aws_connection_info: &aws_connection_info
+      aws_access_key: "{{ aws_access_key }}"
+      aws_secret_key: "{{ aws_secret_key }}"
+      security_token: "{{ security_token }}"
+      region: "{{ aws_region }}"
```

Comment:
I'm guessing you copied this from another test.  Unfortunately we're still in the process of cleaning them up.

It's better to do this using module_defaults:
https://docs.ansible.com/ansible/latest/dev_guide/platforms/aws_guidelines.html#aws-credentials-for-integration-tests

Reply:
I've moved the code to modules_defaults